docs/source/llms/tracing/index.rst

```diff
@@ -690,3 +690,85 @@ Client API
         request_id=parent_span.request_id,
         attributes={"attribute3": "value3", "attribute4": "value4"},
     )
+
+Q: How can I see the stack trace of a Span that captured an Exception?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The MLflow UI does not display Exception types, messages, or stacktraces if faults occur while logging a trace. 
+However, the trace does contain this critical debugging information as part of the Span objects that comprise the Trace. 
+
+The simplest way to retrieve a particular stack trace information from a span that endured an exception is to retrieve the trace directly in 
+an interactive environment (such as a Jupyter Notebook). 
+
+Here is an example of intentionally throwing an Exception while a trace is being collected and a simple way to view the exception details:
+
+.. code-block:: python
+
+    import mlflow
+
+    experiment = mlflow.set_experiment("Intentional Exception")
+
+    with mlflow.start_span(name="A Problematic Span") as span:
+        span.set_inputs({"input": "Exception should log as event"})
+        span.set_attribute("a", "b")
+        raise Exception("Intentionally throwing!")
+        span.set_outputs({"This": "should not be recorded"})
+
+When running this, an Exception will be thrown, as expected. However, a trace is still logged to the active experiment and can be retrieved as follows:
+
+Note that the ``get_trace`` API is **only available** for retrieving traces from the current active session. If you are attempting to retrieve 
+a trace from a previous session, you must use the ``search_traces`` client API (shown below).
+
+
+.. code-block:: python
+
+    trace = mlflow.get_trace(span.request_id)
+    trace_data_spans = trace.data.spans
+    trace_data.to_dict()
```

Comment:
```suggestion
    trace = mlflow.get_trace(span.request_id)
    trace_data_spans = trace.data.spans
    trace_data_spans.to_dict()
```

Reply:
another typo from testing. Ran the code blocks locally and everything's good now.